src/confluent_kafka/src/Consumer.c

```diff
@@ -1409,6 +1436,20 @@ static PyMethodDef Consumer_methods[] = {
           "  :raises: RuntimeError if called on a closed consumer\n"
           "\n"
         },
+        { "memberid", (PyCFunction)Consumer_memberid,
+          METH_VARARGS|METH_KEYWORDS,
+          ".. py:function:: memberid()\n"
+          "\n"
+          " Look up this client's broker-assigned group member id.\n"
+          "\n"
+          " The member id is assigned by the group coordinator and "
+          " is propagated to the consumer during rebalance.\n"
+          "\n"
+          "  :returns: A PyObject with member id or None\n"
```

Comment:
PyObject is a CPython (thus internal) notation, not used in Python docs itself.
Change to: "Member id string or None"

change rtype to string.